### ISSUE TITLE
perl5shared.xml is not populated correctly in multi module perl project

### ISSUE THREAD

**Issue by mciprijanovic** [Camelcade/Perl5-IDEA#1332]
Marking multiple folders as library root in different modules in Perl project that contain more then one module will not populate perl5shared.xml file with all the paths. Paths from just one module are saved in the mentioned file. This leads to the situation that not all paths will be in @INC and not all Perl modules will be available.
Workaround is to edit perl5shared.xml file manually and set all the paths within libRootUrls section in the file (which would be overwritten if again set through the IDE) or to set all the paths in Perl5 parameters in each run configuration. 

Tested on:

Plugin version 2.200
IntelliJ IDEA 2016.2.1
Build #IC-162.1447.26, built on August 2, 2016
JRE: 1.8.0_76-release-b216 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
OS version: CentOS release 6.3 (Final)

**Comment by hurricup** [comment]
You may try http://evstigneev.com/IDEA-Perl5-2.201.zip

**Comment by mciprijanovic** [comment]
I tried. It looks like that saving is now working, but not the including when run specific configuration. In previous version, when I run configuration, paths that were in file perl5shared.xml were passed like parameters to perl. Now, it is not the case. Here are two examples from the console when I run the same configuration in previous and current version:

2.200
/usr/bin/perl -I/home/milosc/var/www/core/lib

2.201
/usr/bin/perl

So, path is not provided as a param. I reverted to version 2.200
